examples/demo_presentation.md

```diff
@@ -0,0 +1,80 @@
+<!--
+This is an example of markdown file that can be played as a presentation with flowery.
+
+To try it, just run:
+
+```
+python -m flowery demo_presentation.md
+```
+-->
+# First Slide
+
+This is a classic slide with a text content.
+
+# Text Formatting
+
+This is a slide with formatting. Text can be **bold** or *italic*.
```

Comment:
Text cannot be rendered with the ~~strike throught~~ tag. This is a limitation of terminals.